### PR TITLE
Rbaff ds migration looker

### PR DESCRIPTION
**Pull request by soGaussian**
Checklist for reviewer:

When adding a new derived dataset:
- [ ] Ensure that the data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data may be available in [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).
- [ ] Avoid merging a PR that includes the logic of a [core metric](https://docs.telemetry.mozilla.org/metrics/index.html) or complex business logic. The recommendation is to implement core business logic in bigquery-etl. E.g. The [type of search](https://github.com/mozilla/bigquery-etl/blob/a3e59f90326816a2ecaaa3e9d5b57fe9552f7d70/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql#L781) or the [calculation of DAU or visited URIs](https://github.com/mozilla/bigquery-etl/blob/9bca48821a8a0d40b1700cc14ecd8068d132ed06/sql/moz-fx-data-shared-prod/telemetry_derived/firefox_desktop_exact_mau28_by_dimensions_v1/query.sql).
- [ ] Avoid merging code in Looker Explores/Views that implement analysis with multiple lines of code or that will be likely replicated in the future. Instead, aim for extending an existing dataset to include the required logic, and use [Looker aggregates](https://cloud.google.com/looker/docs/aggregate_awareness) to facilitate the analysis.
- [ ] Avoid merging a PR with logic that requires validation and health checks. It is recommended to implement it in bigquery-etl for full test coverage and failure alerts.

### CODE REVIEW

File: websites/views/moz_org_metrics_summary.view.lkml [hunk 1]
```diff
@@ -1,6 +1,156 @@
 include: "//looker-hub/websites/views/moz_org_metrics_summary.view.lkml"
 
 view: +moz_org_metrics_summary {
+
+  derived_table: {
```
Comment (kik-kik) [inline websites/views/moz_org_metrics_summary.view.lkml:5]: Could be my limited Looker knowledge, but is this necessary here? We're just doing `SELECT *` here?

We already specify this table here: https://github.com/mozilla/looker-hub/blob/bf36d3cd395aad44729db993e95b42995de40bed/websites/views/moz_org_metrics_summary.view.lkml#LL94C57-L95C2

Does this mean the result of the view will be persisted in a new table or that the properties defined in the Lookml file will be applied on top of it?

File: websites/views/moz_org_metrics_summary.view.lkml [hunk 2]
```diff
@@ -24,4 +180,22 @@ view: +moz_org_metrics_summary {
...
+  measure:  non_fx_downloads_average {
+    label: "Non-Fx Download Average"
+    type: average
+    sql: ${non_fx_downloads} ;;
+  }
+
+  measure:  non_fx_downloads_rate {
+    label: "Non-Fx Download Rate"
+    type: average
+    sql: ${TABLE}.non_fx_download_rate ;;
```
Comment (kik-kik) [inline websites/views/moz_org_metrics_summary.view.lkml:193]: Maybe I missed it, but I don't see `non_fx_download_rate` field in the table?

**Review by kik-kik — APPROVED** [review]
r+, just a couple of comments/questions. Otherwise, LGTM 👍

File: websites/views/moz_org_metrics_summary.view.lkml (hunk 1 above)
Comment (soGaussian) [reply websites/views/moz_org_metrics_summary.view.lkml:5]: The derived table is not necessary but wanted to extend the original table we are creating the view on.

**Pull request merged**